---
title: "Intrinsically disordered region of Talin's FERM domain functions as an initial PIP~2~ recognition site"
date: today
date-format: long
author: #todo
  - name: Jannik Buhr
    url: https://jmbuhr.de
    orcid: 0000-0002-9350-2629
    affiliations:
      - id: hits
        name: Heidelberg Institute for Theoretical Studies
        url: https://www.h-its.org/
      # - id: uni-hd
      #   name: Heidelberg University
      #   url: https://www.uni-heidelberg.de/
      - id: iwr
        name: Interdisciplinary Center for Scientific Computing, Heidelberg University
        url: https://typo.iwr.uni-heidelberg.de/home
    email: 'jannik.buhr@h-its.org'
  - name: Florian Franz
    affiliations:
      - ref: hits
      - ref: iwr
  - name: Frauke Gräter
    url: https://www.h-its.org/people/prof-dr-frauke-grater/
    orcid: 0000-0003-2891-3381
    affiliations:
      - ref: hits
      - ref: iwr
github: https://github.com/hits-mbm-dev/
repo: "paper-talin-loop"
format:
  hits-html:
    css: styles.css
  bj-pdf:
    classoption:
      - twocolumn
  docx: default
colorlinks: true
keep-tex: true
abstract: |
  Focal adhesions mediate the interaction of the cytoskeleton with the extracellular matrix (ECM).
  Cell-ECM adhesion is used by almost all cells both during development and homeostasis and ranges from dynamic to permanent.
  As such, it is an important process in health and disease alike.
  Talin is a central regulator and adaptor protein of the multiprotein focal adhesion complexes and is responsible for integrin activation and force-sensing.
  We evaluated direct interactions of talin with the membrane lipid phosphatidylinositol 4,5-bisphosphate (PIP~2~) by means of molecular dynamics simulations.
  A newly published autoinhibitory structure of talin, where common PIP~2~ interaction sites are covered up, sparked our curiosity for a hitherto less examined loop as a potential site of first contact.
  We show that this unstructured loop in the F1 subdomain of the Talin1 FERM domain is able to interact with PIP~2~ and can facilitate further interactions by serving as a flexible membrane anchor.
  This work presents the dynamics of the interaction and identifies key residues.
  Further, we surveyed the effect of a physiological PIP~2~ enrichment at focal adhesion sites on the dynamics of talin through force-probe molecular dynamics simulations.
  The results provide backing for the direct involvement of PIP~2~ in the localization and activation of talin.
sigstatement: |
  TODO
  Each manuscript must also have a statement of significance or no more than 120 words.
image: ./assets/blender/render/frame0001.png
license: CC BY
keywords:
  - Molecular Dynamics
  - Talin
  - Focal Adhesion
  - Mechanosensing
  - Disorder
  - IDP
  - IDR
citation:
  type: article
  citation-key: buhrTalin2022
  status: draft
editor_options:
  markdown:
    wrap: sentence
editor: source
bibliography: references.bib
---

:::callout-warning
This is a draft and as such subject to change.
:::

::::callout-tip
You can find the poster that goes along with the paper presented at the [Annual Meeting of the Biophysical Society 2022](https://www.biophysics.org/2022meeting#)
in your favorite format here: [web/html](./poster.html), [print/pdf](./poster.pdf).
The source repository for this paper lives [here](https://github.com/hits-mbm-dev/paper-talin-loop).
The preliminary and less feature-rich pdf version lives [here](./index.pdf).
::::

# Introduction

It is critical for cells to mechanically sense their surroundings at cell adhesion sites for a multitude of biological processes.
Contact with the extracellular matrix and surrounding cells regulates growth, differentiation, motility and even apoptosis [@vogelLocalForceGeometry2006; @oakesStressingLimitsFocal2014; @schillerMechanosensitivityCompositionalDynamics2013; @miroshnikovaAdhesionForcesCortical2018].
The multiprotein focal adhesion complex is responsible for translating between biochemical and mechanical signals, where both directions, outside--in and inside--out activation, are being investigated [@thamilselvanPressureActivatesColon2004; @pelletierActivationStateIntegrin1995].

At the center of the focal adhesion complex sits the adaptor protein talin, which dynamically unfolds and refolds under force [@yaoMechanicalResponseTalin2016].
A schematic of Talin can be seen in @fig-tln-schema-long.
Through interaction with integrin tails (dark green) [@tadokoroTalinBindingIntegrin2003], which in turn interact with collagen fibers via their heads, it links the extracellular matrix to the intracellular cytoskeleton by directly interacting with actin.
Talin also features specific interactions with the membrane.
Their formation, mechanical stability and role in mechanosensing remain to be fully resolved.

Talin features an N-terminal FERM domain (F for 4.1 protein, E for ezrin, R for radixin and M for moesin), which is composed of the subdomains F0 to F3 and provides a link to the cystosolic side of the plasma membrane [@chishtiFERMDomainUnique1998].
It does so via a conserved binding motif for phosphatidylinositol 4,5-bisphosphate (PIP~2~), which is enriched at active focal adhesion sites [@maniFERMDomainPhosphoinositide2011; @dasMolecularMechanotransductionPathway2015; @songNovelMembranedependentSwitch2012a].
The main PIP~2~ binding sites are located in F2 and F3 (highlighted as red spheres in @fig-tln-schema-long).
Notably, the Talin1 FERM domain differs from other FERM proteins through the addition of the F0 subdomain, which is connected to F1 via a charged interface, as well as an insertion in F1, a flexible loop with helical propensity and basic residues [@goultStructureDoubleUbiquitinlike2010].

Additionally, Talin's FERM domain exists in an extended conformation, as opposed to the cloverleaf-like conformation of other FERM proteins [@elliottStructureTalinHead2010].
F3 also has a binding site for $\beta$-integrin tails [@calderwoodTalinHeadDomain1999] and is partly responsible for the enrichment of PIP~2~ at the membrane through a binding site for PIPKI$\gamma$ [@calderwoodTalinsKindlinsPartners2013].
A second integrin binding site is located with the rod domain 11 (R11) [@horwitzInteractionPlasmaMembrane1986].
Talin interacts with the cytoskeleton through actin binding sites (F2-F3, R4-R8, R13-DH) [@mccannLWEQModuleConserved1997].
The review by Klapholz et al. [@klapholzTalinMasterIntegrin2017] provides an excellent overview of the many interaction sites of talin and its central role in the focal adhesion complex.

The mechanistic role of the disordered F1 loop in the many aspects of Talin function remains elusive.
Its overall positive charge renders it a prime candidate as a PIP~2~ binding site.
However, previous studies only identified a minor role of the loop in PIP~2~ binding compared to F2-F3 [@chinthalapudiInteractionTalinCell2018a; @saltelNewPIP22009].
On the other hand, the F1 loop has been shown to contribute to talin-mediated integrin activation [@goultStructureDoubleUbiquitinlike2010].

<!-- TODO: paragraph on FERM-PIP2 at focal adhesions, -->
<!-- multiple binding sites, several pip2s critial for stability of FAK, -->
<!-- allows opening of autoinhibited FAK -->

It was previously shown that F3 can interact with R9, which impedes integrin activation [@bannoSubcellularLocalizationTalin2012].
Furthermore, in a recently determined cryo-electron microscopy structure of autoinhibited Talin1, Dedden et al. [@deddenArchitectureTalin1Reveals2019a] showed that the rod domains R9 and R12 shield the established PIP~2~ binding surface and the integrin binding site in F3 (see [@fig-tln-schema-autoinhib; @fig-tln-align-autoinhib]).
This beckons the question how this autoinhibition can be resolved.
Song at al. [@songNovelMembranedependentSwitch2012a] previously investigated a fragment of talin consisting of F2-F3 and an inhibiting rod segment and suggested a pull-push mechanism, whereby negatively charged PIP~2~ attracts its positively charged binding surface on F2-F3 and simultaneously repels the negatively charged surface of the inhibitive rod segment.
However, this still leaves open the question of how talin can establish a first contact with the membrane and remain within a sufficient proximity for this effect to kick in.

::::{#fig-structure layout="[[1], [1,1], [1]]"}

![ ](./assets/blender/render/frame0000.png){#fig-tln-schema-long}

![ ](./assets/blender/render/frame0001.png){#fig-tln-schema-autoinhib}

![ ](./assets/blender/render-align/frame0000.png){#fig-tln-align-autoinhib}

A schematic overview of Talin and our simulation setup.
**a\)**
A schematic rendering of full-length Talin over a POPC membrane enriched with PIP~2~ in the upper leaflet.
The subdomains under scrutiny in this publication, namely F0-F3, which comprise the N-terminal FERM domain (or talin head) are highlighted in pastel colors (green, cyan, yellow, magenta).
The two major PIP~2~ binding sites in F2-F3 are marked with red spheres.
An integrin binding site also resides in this area.
Klapholz et al. [@klapholzTalinMasterIntegrin2017] provide an excellent review of Talin's central role in the focal adhesion complex and list further binding sites.
The talin rod segments (or talin tail) are numbered R1 to R13.
Note that under physiological conditions, with talin experiencing force from bound actin, the angle between the FERM domain and the talin rod would be more akin to 30° as opposed to the linear structure shown here for illustrative purposes.
Tails of an integrin $\alpha$ and $\beta$ heterodimer reaching through the lipid bilayer are represented in green.
**b\)**
A schematic rendering of the autoinhibited structure of Talin as crystallized by Dedden et al. [@deddenArchitectureTalin1Reveals2019a] in combination with a cartoon representation in **c)**.
The completed FERM structure by Elliott et al. [@elliottStructureTalinHead2010], with our addition of the modelled F1 loop, is fitted to the autoinhibited structure, as the latter does not include F0-F1 due to their flexibility.
The complete FERM structure can be explored interactively in the context of our simulation system in @sec-system.
The main PIP~2~-binding sites in F2-F3 are occluded by rod domain 12 Additionally, R9 covers up the integrin binding site.
::::

Here we test the hypothesis that the flexible F1 loop inserted into Talin's FERM domain serves as an additional PIP~2~ interaction site that is readily accessible to PIP~2~ even in Talin's autoinhibited conformation and further mechanically stabilizes Talin's interaction with the membrane.
To this end, we modelled the loop, which due to its high flexibility, is not included in crystal structures of the FERM domain, such as PDB-ID 3IVF by Elliot et al. [@elliottStructureTalinHead2010].

With a complete structure of the talin FERM domain we investigated the role of the F1 loop through molecular dynamics simulations, which had previously also proven useful to detect the recognition of PIP~2~ in membranes by PH domains [@buyanMultiscaleSimulationsSuggest2016] or the FERM domain of Focal Adhesion Kinase [@zhouMechanismFocalAdhesion2015].

In F0-F1 simulations, we found the loop to have a clear propensity to interact with the PIP~2~-containing membrane.
Moreover, it is to be able to establish a first contact with the membrane even from unfavorable initial orientations due to its large search volume.
Furthermore, we show with simulations of the full-length FERM domain that once the loop has established an initial contact, it can anchor the FERM domain to the membrane and establish the known major binding sites in F2-F3.

These results provide mechanistic insight into the role of membrane interactions for the localization of Talin at the center of the focal adhesion complex and highlight the role of secondary flexible binding surfaces for membrane recognition.

# Materials and Methods

## Molecular dynamics with GROMACS

Molecular dynamics simulations were performed with GROMACS [@berendsenGROMACSMessagepassingParallel1995; @abrahamGROMACSHighPerformance2015] version 2020.03 [@lindahlGROMACS2020Source2020].
A crystal structure of the Talin FERM domain by Elliot et al. [@elliottStructureTalinHead2010] with the PDB-ID 3IVF was used as the basis of all simulations.

The missing F1 domain loop between residues L133 and W144 was modeled using MODELLER [@marti-renomComparativeProteinStructure2000; @webbComparativeProteinStructure2016] via the interface to Chimera [@pettersenUCSFChimeraVisualization2004], followed by equilibration with GROMACS.
The resulting conformation (see @sec-system) was compared to an NMR structure of the F1 domain (PDB-ID 2KC2) by Goult et al. [@goultStructureDoubleUbiquitinlike2010].

The missing residue M1 was also added.
The missing residues I399 and L400 were not modeled, leaving us with with a continuous sequence from residue 1 to 398.
Simulations were performed with the CHARM36 force field.
Topologies, including the membrane, were generated with the CHARM-GUI web app [@brooksCHARMMBiomolecularSimulation2009; @joCHARMMGUIWebbasedGraphical2008; @leeCHARMMGUIInputGenerator2016] and GROMACS tools.
All simulations used the TIP3P water model and were neutralized with 0.15 mol/L of NaCl A 6-step equilibration was performed after gradient decent energy minimization while gradually relieving restraints on protein and membrane atoms Production runs use a timestep of 2 fs, a Verlet cut-off scheme for Van-der-Waals interactions and the Particle Mesh Ewald (PME) method for long-range electrostatics.
NTP-ensembles were achieved by Nosé-Hoover temperature coupling [@hooverCanonicalDynamicsEquilibrium1985; @noseUnifiedFormulationConstant1984] and Parinello-Rahman pressure coupling [@parrinelloPolymorphicTransitionsSingle1981].
An example `.mdp`-file can be found in the supplementary materials in @sec-prod-mdp.

The initial equilibrium simulation of the completed FERM domain was run for 75 ns.
Subsequently, the root mean squared fluctuation (RMSF) was calculated with GROMACS tools.

The F0F1 FERM sub domains (residues 1 to 197) were simulated to evaluate protein-membrane association using a rotational sampling approach.
This entailed placing the protein 1.5 nm away from a 1-palmitoyl-2-oleoyl-glycero-3-phosphocholine (POPC) membrane in a total of 60 orientations spanning a rotation of 360 degrees.
6 replicates of each orientation were run for 200\ ns each.
However, due to a hardware failure, 6 of these 360 runs are only 50 to 150\ ns long.
Of the 119 lipids in the upper leaflet of the POPC membrane, 12 lipids were replaced with PIP~2~, which results in a physiological concentration of 10% PIP~2~.

From this rotational sampling, we selected representative conformations with loop-membrane interactions as the basis of 6 equilibrium simulations of the complete FERM domain over a POPC membrane with 26 PIP~2~ lipids out of a total of 273 lipids in the upper leaflet.
Each simulation ran for 400\ ns.
The initial conformations for perpendicular pulling simulations of the F0F1 subdomains to gauge interaction strength were also chose from the rotational sampling set.

Distance information was extracted from trajectories with gromacs tools interfaced via CONAN [@mercadanteCONANToolDecode2018].

## Automation, Data Analysis and Availability

Setup scripts written in bash are available for all simulations shown in this work.
Computations for data analysis were tracked with the targets R package [@landauTargetsPackageDynamic2021].
Plots were generated with ggplot2 [@ggplot].
Interactive structure representations are embedded using Mol\* [@molstar].
Schematic visualizations were rendered with blender [@blender] and VMD [@vmd].
Files relevant to this paper that are too big to be uploaded to this repository, such as trajectories and blender files, will be uploaded to a separate location.
This paper and the matching poster were generated with [quarto](https://quarto.org/) [@quarto; @knitr; @rbetterposter].

::: {.content-visible when-format="html"}
The plotting code and computational graph of `targets` is available in [this document](./_analysis.html).
:::

# Results

## The F1 loop can act as a point of first contact

The high flexibility of the F1 loop gave use the confidence to model it from sequence It retained its flexibility in equilibrium simulations (@fig-loop-rmsf), which in combination with comparisons to NMR structures [@goultStructureDoubleUbiquitinlike2010] confirmed this approach.
The resulting system that provides the basis for our simulations can be explored interactively in @sec-system.

When simulating only F0-F1 over a POPC membrane containing 10% PIP~2~, we noticed that the F1 loop had a clear propensity to establish contact with the membrane.
And once contact had been established the protein was anchored strongly enough for more contacts to evolve with time, pulling the protein onto the membrane ([see Figure @fig-f0f1-unbound; -@fig-f0f1-anchored; -@fig-f0f1-bound]).
In order to control for a potential bias towards the loop as a result of the starting position we performed a rotational sampling of the system, where the starting angle of the loop with respect to the membrane was varied across 60 equally spaced angles.
@fig-f0f1-ri-angle shows that independent of the starting position, the loop is able to find the membrane and bind to it, though this does happen earlier in the simulation when the loop starts favorably oriented towards the membrane (@fig-f0f1-angle-frame).
However, even in its most unfavorable starting orientation (180°, oriented away from the membrane) the loop is able to find the membrane due to the large search space it can cover with its high flexibility (see @fig-loop-rmsf).

Once contact has been made, it becomes exceedingly unlikely for F0-F1 to dissociate from the membrane (@fig-f0f1-retention).
Of 358 runs ^[6 replicas each for 60 angles minus 2 runs lost to a storage failure], 89 runs never made contact with the membrane, but out of the 269 that did, only 10 eventually dissociated.

::: {.content-visible when-format="html"}
@fig-f0f1-ri-npip and @fig-f0f1-residues highlight the residues involved in the interaction.
:::

::: {.content-visible when-format="pdf"}
@fig-f0f1-ri-npip highlights the residues involved in the interaction.
:::

::::: {#fig-loop-importance layout="[[1,1,1],[1]]"}

![ ](./assets/vmd/f0f1/unbound.png){#fig-f0f1-unbound}

![ ](./assets/vmd/f0f1/anchored.png){#fig-f0f1-anchored}

![ ](./assets/vmd/f0f1/bound.png){#fig-f0f1-bound}

![ ](./results/plots/f0f1-ri-angle-npip-1.png){#fig-f0f1-ri-angle}

Rotational sampling of F0-F1.
**a-c\)**
Snapshots from a simulation involving F0-F1 over a POPC membrane containing 10% PIP~2~ in the upper leaflet.
POPC is not rendered and PIP~2~ is shown as light grey stick models that turn thicker for those molecules that are currently interacting with residues of the protein.
Those residues are then shown as dark blue stick models.
Once the F1 loop has made contact with the membrane it can act as an anchor and facilitate further contacts, ultimately pulling the protein onto the membrane.
In order to test if this interaction of the loop with the membrane is just the result of a biased starting position with the loop already pointing downwards, we sampled 60 different starting positions, rotated equally spaced around the horizontal axis, with 6 replicates each.
**d\)**
A heatmap summarizing 358 simulations from the rotational sampling.
Unfortunately the number is not 360 because 2 trajectories were lost due to a hardware failure.
Each simulation is 200\ ns long.
Across all angles (y-axis) we see that with very few exceptions the F1 loop (dark blue region on the x-axis colorbar) is almost always involved in interactions 0° equates to the loop pointing downwards towards the membrane.
The heatmap color represents the mean number of PIP~2~ molecules that are in close contact with the respective residue summarized over time and replicates for that specific angle.
:::::


::::: {#fig-loop-importance layout="[[1,1]]"}
![ ](./results/plots/f0f1-angle-frame-npip-1.png){#fig-f0f1-angle-frame}

![ ](./results/plots/f0f1-retention-1.png){#fig-f0f1-retention}

**a\)**
A heatmap of the time evolution of the number of PIP~2~ molecules at the respective time and angle summarized over all residues and replicates.
Angles in which the loop is already favored towards the membrane tend to make contact faster.
Note that this trend is not simply because angles favoring the loop would have been already closer to the membrane.
The protein was rotated in such a way that the respective closest residue had the same distance to the membrane for the 0° and the 180° starting positions.
**b\)**
A time evolution of the simulations shows the number of interacting residues gradually increasing as the anchored protein gets pulled closer towards the membrane by the forming interactions.
Only in 10 simulations (out of 269 simulations that had at least one contact) did the protein leave the membrane again within the 200\ ns long timeframe.
This never occurred after more than 3 residues had already made contact.
:::::

::::: {#fig-f0f1-residues layout="[[1],[1]]"}

![ ](./results/plots/f0f1-ri-npip-1.png){#fig-f0f1-ri-npip}

:::{#fig-f1f1-residues}
::: {.content-visible when-format="html" }
![](./assets/vmd/f0f1/residues.mp4)
:::
:::{.content-visible when-format="pdf" width=20%}
![](./assets/vmd/f0f1/residues.png)
:::
 
:::

PIP~2~-interacting residues of F0-F1.
**a\)**
The Mean interaction scores of the individual residues across all simulations that made contact with the membrane.
Color represents the isoelectric point of the amino acid in isolation (blue = basic, magenta = acidic).
A number of very prominent lysines can be observed, as well as a cluster of residues belonging to the F1 loop.
The most prominent residues are highlighted in **b)**.
(For the print version it is just a placeholder image. The video is available in the web-version (<https://hits-mbm-dev.github.io/paper-talin-loop>) or here: <https://youtu.be/s5yya0XeNTA>).
:::::

## The F1 loop can facilitate further contacts

We chose a representative conformation from the rotational sampling as a starting point for force-probe simulations of F0-F1 perpendicular to the membrane  to test the strength of the interaction (@fig-f0f1-vert-pull-vmd).
An exemplary render of one of the simulations can be seen in @fig-f0f1-pull-run-1.
Pulling F0-F1 off the membrane requires peak forces of 100--120\ pN, during which the interacting residues only very gradually loose contact (@fig-f0f1-vert-pull).
This highlights the strong anchoring capabilities of the F1 loop.
As seen in @fig-f0f1-vert-pull-contacts, during pulling residues not belonging to the F1 loop loose contact first, while the loop stays attached.
The F1 loop works in conjunction with the F0 subdomain (see @fig-f0f1-vert-pull-residues).
Their high flexibility allows them to remain in contact with the membrane over large distances, which would allow for a spring-like re-establishing of more contacts should the force be alleviated.

:::: {#fig-vert-pull layout="[[1.6, 1],[1, 1]]"}

:::{#fig-f0f1-pull-run-1}
::: {.content-visible when-format="html"}
![](./assets/vmd/f0f1-pulling/f0f1-run1.mp4)
:::
::: {.content-visible when-format="pdf"}
![](./www/hits-logo-small.png)
:::
 
:::

![ ](./assets/vmd/f0f1-pulling/vert-pull.png){#fig-f0f1-vert-pull-vmd}

![ ](./results/plots/f0f1-vert-pull-1.png){#fig-f0f1-vert-pull}

![ ](./results/plots/f0f1-vert-pull-contacts-time-1.png){#fig-f0f1-vert-pull-contacts}

Vertical Pulling of F0-F1.
**a\)**
A representative render of one of 6 force-probe MD simulations pulling F0-F1 off the membrane (For the print version it is just a placeholder image. The video is available in the web-version (<https://hits-mbm-dev.github.io/paper-talin-loop> or here: <https://youtu.be/-eZ2orx7QRE>).
It starts from a snapshot of F0-F1 in its bound conformation taken from the rotational sampling (@fig-loop-importance) and gets pulled upwards from its C-terminus.
The direction of force is shown in the snapshot **b\)**.
**c\)**
As F0-F1 gets pulled at a constant rate of 0.03\ nm/ns we observe the time evolution of the force (bottom panel) and the number of interacting residues (top panel).
The number of interacting residues goes down very gradually, as the high flexibility of loop allows the residues to remain in contact even as the distance increases.
Replicate 4 is highlighted in magenta, as in this run the interactions were so so strong that a total of 3 molecules of PIP~2~ were pulled out of the membrane (1 by F0 and 2 by the F1 loop).
A snapshot of this can be seen in @fig-f0f1-vert-pull-run4.
**d\)**
The time evolution of the number of contacts for resides belonging to the F1 loop and other residues shows how initially other residues loose contact until eventually the loop looses contacts as well.
Lighter shades of blue correspond to a later time in the simulation.
Black dots mark the starting positions.
The longest remaining non-loop contacts belong to the N-terminus of F0 (for which **b** is also a representative snapshot), with the exception of replicate 4, as explained in **c**.
::::

Simulations with the full-length FERM domain show that with the loop as an initial membrane contact, known PIP~2~ binding sites can also be established ([@fig-ferm-ri-npip; @fig-ferm-memb-system]).
The highlighted residues include K272 of F2 and K316, K324, E342, and K343 of F3, which have been shown to be crucial for the membrane interaction of Talin and subsequent integrin activation by Chinthalapudi et al. [@chinthalapudiInteractionTalinCell2018a].

:::: {#fig-ferm-further layout="[[1],[1]]"}

![ ](./results/plots/ferm-ri-npip-1.png){#fig-ferm-ri-npip}

![ ](./assets/vmd/ferm/ferm-residues-transparent-arrows.png){#fig-ferm-memb-system}

Simulation of the full-length FERM domain over a 10% PIP~2~-membrane.
**a\)**
The Mean interaction scores of the individual residues across 6 simulations.
Color represents the isoelectric point of the amino acid in isolation (blue = basic, magenta = acidic).
The known PIP~2~ interaction sites K272 of F2 and K316, K324, E342, and K343 of F3 [@chinthalapudiInteractionTalinCell2018a] are highlighted with red lines on the x-axis colorbar and can also be seen in the cartoon representation in **b\)** where the main interacting residues are displayed as dark blue stick models.
::::

# Discussion and Outlook

Using molecular dynamics simulations, we provide mechanistic insight into the membrane recognition dynamics of Talin.
This adds a new mode of interaction that helps to explain how Talin can find the membrane even when its main PIP~2~ (and integrin) binding sites in F2 and F3 [@chinthalapudiInteractionTalinCell2018a] (see figure @fig-ferm-memb-system ) are blocked by autoinhibition [@deddenArchitectureTalin1Reveals2019a].
This interaction mode is not characterized by strong binding sites interacting with one molecule of PIP~2~ each, as would be the conclusion from crystallographic data alone.
Rather the cumulative diffuse interaction of multiple PIP~2~ with multiple residues is what keeps the protein anchored to the membrane.
This is particularly evident in the interaction with the flexible F1 loop, but also in the F0 domain.
While crystal structures of proteins in complex with PIP~2~ typically show a one-to-one ratio of lipid per binding site [@chinthalapudiLipidBindingPromotes2018; @chinthalapudiInteractionTalinCell2018a; @jianMolecularBasisCooperative2015], possibly due to the nature of the experimental method, our simulations suggest multiple PIP~2~ molecules binding simultaneously.
Similar results have been observed for Pleckstrin Homology (PH) domain proteins by Naughton et al. [@naughtonModesInteractionPleckstrin2018].
According to their study, this simultaneous binding of multiple PIP~2~ molecules contributes to the high affinity of the membrane interaction.

Our molecular dynamics simulations suggest that the F1 loop can find favorable interactions with PIP~2~ across large distances in a large search volume due to its flexibility.
A similar mechanism has also been shown by Shoemakter et al. [@shoemakerSpeedingMolecularRecognition2000] and was fittingly coined "fly-casting".
In the aforementioned publication they focus on the interaction of unfolded regions with DNA.
Our simulations now provide an example for the concept applied to protein-lipid interactions.
It its well worth noting that, although we mention the greater search space of the F1 loop as its advantage in recognizing PIP~2~, it has also been argued that the kinetic advantage of the fly-casting mechanism comes mainly from the reduction in free energy as the disordered region folds around the interaction target [@huangKineticAdvantageIntrinsically2009].
The fast binding kinetics are crucial for Talin's function at focal adhesion sites.
As the PIP~2~ concentrations increases at the active focal adhesion site, Talin's FERM F1 loop can perform a quick recognition.
The flexibility of the loop also allows it to anchor the protein at the membrane even when being stretched under force (up to a delta of 7 nm, as seen in @fig-f0f1-vert-pull).
This is akin to the elastic response seen in focal adhesion kinase (FAK) under force, in which a 49 AA linker allows for buffering of the force [@bauerStructuralMechanisticInsights2019].
In our force probe experiments we pulled F0F1 orthogonally off of the membrane.
This was useful in showing the full extension and force resistance of the loop.
_In vivo_, however, Talin's FERM domain is subjected to forces acting at a 30° angle.
This might imply an additional function for the FERM domain.
As it is dragged along the membrane, the diffuse interactions of the F1 loop and main interaction sites in F2-F3 with PIP~2~ would increase lateral friction along the membrane as the PIP~2~ concentration increases.
This could further localize Talin at active focal adhesion sites.

We conclusively show that the F1 loop is able to interact with the membrane even from most unfavorable positions.
We propose that talin mutants lacking the loop, or specifically the basic residues in said loop, will show reduced or at least slowed-down focal adhesion maturation, increased lateral diffusion of talin under force and faster focal adhesion disassembly.

But recognition is only the first step.
It would indeed be fascinating to also provide mechanistic ideas for the resolution of the autoinhibition by all-atom simulations of the FERM domain that also include an inhibiting rod segment.
These larger-scale simulations might then be able to provide evidence for the push--pull mechanism proposed by Song et al. [@songNovelMembranedependentSwitch2012a] or result in novel ideas.
More generally, we propose positively charged, intrincsically disordered regions in PIP~2~-binding domains to promote recognition and help to maintain the interaction under force, in FERM domains and elsewhere.

# Author Contributions

Conceived and designed the experiments:\ JB FF FG.
Performed the experiments:\ JB FF.
Analyzed the data:\ JB FF.
Contributed reagents/materials/analysis tools:\ FF.
Wrote the paper:\ JB.

# Acknowledgments

This project has received funding from the European Research Council (ERC) under the European Union’s Horizon 2020 research and innovation programme (grant agreement No. 101002812).

This work was supported by the Klaus Tschira Foundation.

# References

:::{#refs}
:::

# Supplementary Material

## Simulation System {#sec-system}

:::: {.content-visible when-format="pdf"}
:::callout-note
This interactive display is only available in the web version: <https://hits-mbm-dev.github.io/paper-talin-loop/>
:::
::::

:::: {.content-visible when-format="html"}
:::{#app2 .app .video-container}
:::

<script type="text/javascript" src="./www/molstar.js"></script>
<link rel="stylesheet" type="text/css" href="./www/molstar.css" />
<script type="text/javascript">
var viewer2 = new molstar.Viewer('app2', {
  layoutIsExpanded: false,
  layoutShowControls: false,
  layoutShowRemoteState: false,
  layoutShowSequence: true,
  layoutShowLog: false,
  layoutShowLeftPanel: true,
  viewportShowExpand: true,
  viewportShowSelectionMode: false,
  viewportShowAnimation: false,
  pdbProvider: "rcsb",
  emdbProvider: "rcsb",
});
viewer2.loadStructureFromUrl('./www/system-tln-memb.pdb', format='pdb')
</script>
::::

## Scripts

:::callout-note
Analysis scripts, setup scripts and production trajectories will be uploaded and linked here.
:::


## Supplementary Plots and Tables

::: {#fig-suppl layout="[[1,1],[1,1],[1]]"}

![ ](./assets/results/figures/loop-rmsf.png){#fig-loop-rmsf}

![ ](./results/plots/f0f1-distance-cutoff-1.png){#fig-r-hist}

![ ](./results/plots/ferm-time-ri-npip-all-1.png){#fig-ferm-time-ri-npip-all}

![ ](./results/plots/f0f1-vert-pull-residues-1.png){#fig-f0f1-vert-pull-residues}

![ ](./assets/vmd/f0f1-pulling/snapshot-run4.png){#fig-f0f1-vert-pull-run4}


**a\)**
RMSF [nm] of the c$\alpha$ of individual residues in an equilibrium simulation shown by coloring the backbone The loop is highly flexible.
**b\)**
A density plot of distances between PIP~2~ and the protein residues to decide on a cutoff for defining interactions A distance of 0.25 nm was chosen.
**d\)**
A closer look at the residues involved in the interaction during pulling reveals the instrumental role of both the F1 loop as well as the F0 subdomain in keeping the connection to the membrane.
**e\)**
Run 4 of the vertical pulling of F0F1. Interactions between the protein and PIP~2~ were so strong that a total of 3 molecules of PIP~2~ (gray) were pulled out of the membrane (1 by F0 (green) and 2 by the F1 loop (blue)).
:::

Table: Top residues interacting with F0F1 {#tbl-f0f1-top-interacting}

|Residue | Mean #PIP~2~|
|:-------|----------:|
|M 1     |      0.188|
|K 15    |      0.184|
|R 30    |      0.173|
|R 35    |      0.245|
|R 74    |      0.124|
|K 98    |      0.176|
|R 118   |      0.209|
|T 144   |      0.299|
|L 145   |      0.168|
|K 147   |      0.263|
|L 151   |      0.325|
|D 154   |      0.248|
|E 155   |      0.261|
|M 158   |      0.272|
|K 160   |      0.254|
|K 162   |      0.181|
|L 193   |      0.200|
|R 194   |      0.101|

Table: Top residues interacting with FERM {#tbl-ferm-top-interacting}

|Residue | Mean #PIP~2~|
|:-------|----------:|
|M 1     |      0.118|
|T 144   |      0.322|
|L 145   |      0.129|
|K 147   |      0.412|
|L 151   |      0.204|
|D 154   |      0.293|
|E 155   |      0.257|
|M 158   |      0.246|
|K 160   |      0.459|
|K 162   |      0.195|
|Y 270   |      0.156|
|K 272   |      0.180|
|G 275   |      0.222|
|L 314   |      0.141|
|K 316   |      0.304|
|K 318   |      0.148|
|K 320   |      0.552|
|G 321   |      0.174|
|K 322   |      0.442|
|D 341   |      0.142|
|S 362   |      0.363|


### Molecular Dynamics Parameters {#sec-prod-mdp}

```bash
integrator              = md
dt                      = 0.002
nsteps                  = 100000000
nstxout                 = 5000
nstvout                 = 5000
nstfout                 = 50000
nstcalcenergy           = 100
nstenergy               = 1000
nstlog                  = 1000
cutoff-scheme           = Verlet
nstlist                 = 20
rlist                   = 1.2
coulombtype             = pme
rcoulomb                = 1.2
vdwtype                 = Cut-off
vdw-modifier            = Force-switch
rvdw_switch             = 1.0
rvdw                    = 1.2
tcoupl                  = Nose-Hoover
tc_grps                 = SYSTEM
tau_t                   = 1.0
ref_t                   = 303.15
pcoupl                  = Parrinello-Rahman
pcoupltype              = semiisotropic
tau_p                   = 5.0
compressibility         = 4.5e-5  4.5e-5
ref_p                   = 1.0     1.0
constraints             = h-bonds
constraint_algorithm    = LINCS
continuation            = yes
nstcomm                 = 100
comm_mode               = linear
comm_grps               = SYSTEM
refcoord_scaling        = com
```

